sirepo/feature_config.py

```diff
@@ -9,48 +9,41 @@
 
 #: Codes that depend on other codes. [x][0] depends on [x][1]
 _DEPENDENT_CODES = [
-    ['jspec', 'elegant'],
-    ['controls', 'madx'],
+        ['jspec', 'elegant'],
+        ['controls', 'madx'],
+
 ]
 
 #: Codes on prod
 PROD_FOSS_CODES = frozenset((
-    'controls',
-    'elegant',
-    'jspec',
-    'madx',
-    'ml',
-    'opal',
-    'radia',
-    'shadow',
-    'srw',
-    'synergia',
-    'warppba',
-    'warpvnd',
-    'zgoubi',
+        'controls',
```

Comment:
indentation should be 4 not 8. It counts on the previous line. Just because there are two parens, doesn't mean you need to increase indentation, rather a good thing not to in this case.

Reply:
Sorry, copy paste bug from codes.py cf155829d